ros2_batch_job/packaging.py

```diff
@@ -24,33 +24,37 @@
 
 
 def build_and_test_and_package(args, job):
-    print('# BEGIN SUBSECTION: ament build')
+    print('# BEGIN SUBSECTION: build')
```

Comment:
this is "BEGIN SUBSECTION: colcon build" in the other file. I suggest homogenizing by just using "build", "test" and "test-result" everywhere

Reply:
Done in 78d6cab9de6dcd9ca098875a175f78f6604e2975.